client/kit/src/routes/login/+page.svelte

```diff
@@ -26,7 +26,9 @@
         secure: process.env.NODE_ENV === "production",
       });
 
-      await gotoSapper("/");
+      // TEMP SOLUTION TO MAKE IT WORK LOCALLY
+      // DO NOT KEEP THIS IS BAD
+      await gotoSapper("http://localhost:3002");
```

Comment:
Same here, as long as I can't make the client proxy work I had to add the domain on which my local works